devsetup/scripts/tripleo.sh

```diff
@@ -248,6 +243,9 @@ if [[ "$EDPM_COMPUTE_CEPH_ENABLED" == "true" ]]; then
 fi
 
 if [[ -f $HOME/containers-prepare-parameters.yaml ]]; then
+    echo "Using existing containers-prepare-parameters.yaml - contents:"
+    # requires 'make download_tools'
+    yq '.parameter_defaults.ContainerImageRegistryCredentials="{ ...snip... }"' $HOME/containers-prepare-parameters.yaml
```

Comment:
ok i see this should address both my comments above, one about the echo and the other about the credentials. As long as we keep setting the user/pass to "" for the ci jobs case (the actual login happens with zuul secrets in config repo)